modules/ROOT/pages/_partials/sgw-svr-compatibility.adoc

```diff
@@ -226,6 +238,40 @@ Compatibility Matrix
 | image:ROOT:yes.png[]
 | image:ROOT:yes.png[]
 | image:ROOT:yes.png[]
+| image:ROOT:yes.png[]
+
+| 4.0.0
+| 
+| image:ROOT:no.png[]
+| image:ROOT:no.png[]
+| image:ROOT:no.png[]
+| image:ROOT:yes.png[]
+| image:ROOT:yes.png[]
+| image:ROOT:yes.png[]
+| image:ROOT:yes.png[]
+| image:ROOT:yes.png[]
+
+| 4.0.0
+| Using Scopes and Collections
+| image:ROOT:no.png[]
+| image:ROOT:no.png[]
+| image:ROOT:no.png[]
+| image:ROOT:no.png[]
+| image:ROOT:yes.png[]
+| image:ROOT:yes.png[]
+| image:ROOT:yes.png[]
+| image:ROOT:yes.png[]
+
+| 4.0.0
+| Using Version Vectors
```

Comment:
Do not use Version Vectors in public document. This is required for Active-Active XDCR compatibility.

I'm not 100% sure that the public facing terminology is Active-Active XDCR, use whatever words are requried.